truffle/src/main/java/org/jruby/truffle/nodes/ext/BigDecimalNodes.java

```diff
@@ -1245,55 +1303,31 @@ public double toFSpecial(RubyBasicObject value) {
                 case NAN:
                     return Double.NaN;
                 default:
-                    throw new RuntimeException();
+                    CompilerAsserts.neverPartOfCompilation();
+                    throw new UnsupportedOperationException();
             }
         }
 
     }
 
-    @CoreMethod(names = "split")
-    public abstract static class SplitNode extends BigDecimalCoreMethodNode {
-
-        final private ConditionProfile wasNormal;
-        @Child private CallDispatchHeadNode signCall;
-        @Child private CallDispatchHeadNode exponentCall;
-        @Child private ToIntNode signToInt;
-        @Child private ToIntNode exponentToInt;
-        @Child private IntegerCastNode signIntegerCast;
+    @CoreMethod(names = "unscaled")
```

Comment:
If this method is not present in normal Ruby, it should be `private` (if possible or `protected` otherwise) and annotated with `@RubiniusOnly` (not so well named for this).